Core/Properties.cpp

```diff
@@ -75,49 +106,45 @@ void ReadParticles(PartList &list,
     // cparity is optional
     double cparity = 0.0;
     try {
-      cparity = tmp.GetQuantumNumber("Cparity");
+      cparity = tmp.getQuantumNumber("Cparity");
     } catch (std::exception &ex) {
     }
 
-    LOG(DEBUG) << "ReadParticles() | Particle " << tmp.name()
-               << " (id=" << tmp.GetId() << ") "
-               << " J(PC)=" << tmp.GetSpinQuantumNumber("Spin") << "("
-               << tmp.GetQuantumNumber("Parity") << cparity << ") "
-               << " mass=" << tmp.GetMass()
-               << " decayType=" << tmp.GetDecayType();
+    LOG(DEBUG) << "ReadParticles() | Particle " << tmp.getName()
+               << " (id=" << tmp.getId() << ") "
+               << " J(PC)=" << tmp.getSpinQuantumNumber("Spin") << "("
+               << tmp.getQuantumNumber("Parity") << cparity << ") "
+               << " mass=" << tmp.getMass().Value
+               << " decayType=" << tmp.getDecayType();
   }
 
   return;
 }
 
-
-
-boost::property_tree::ptree ParticleProperties::Save(){
+/*boost::property_tree::ptree ParticleProperties::save() {
   boost::property_tree::ptree pt;
-  pt.put("<xmlattr>.Name",Name);
+  pt.put("<xmlattr>.Name", Name);
   pt.put("Pid", Id);
-  pt.add_child("Parameter",Mass.save());
-  pt.put("Parameter.<xmlattr>.Type","Mass");
-  for( auto& i: spinQuantumNumbers_ ){
+  pt.put("Mass", Mass);
+  for (auto &i : spinQuantumNumbers_) {
     boost::property_tree::ptree tmp;
-    tmp.put("<xmlattr>.Class","Spin");
-    tmp.put("<xmlattr>.Type",i.first);
-    tmp.put("<xmlattr>.Value",i.second.GetSpin());
-    pt.add_child("QuantumNumber",tmp);
+    tmp.put("<xmlattr>.Class", "Spin");
+    tmp.put("<xmlattr>.Type", i.first);
+    tmp.put("<xmlattr>.Value", i.second.GetSpin());
+    pt.add_child("QuantumNumber", tmp);
   }
-  for( auto& i: intQuantumNumbers_ ){
+  for (auto &i : intQuantumNumbers_) {
     boost::property_tree::ptree tmp;
-    tmp.put("<xmlattr>.Class","Int");
-    tmp.put("<xmlattr>.Type",i.first);
-    tmp.put("<xmlattr>.Value",i.second);
-    pt.add_child("QuantumNumber",tmp);
+    tmp.put("<xmlattr>.Class", "Int");
+    tmp.put("<xmlattr>.Type", i.first);
+    tmp.put("<xmlattr>.Value", i.second);
+    pt.add_child("QuantumNumber", tmp);
   }
-  pt.add_child("DecayInfo",DecayInfo);
+  pt.add_child("DecayInfo", DecayInfo);
   return pt;
+}*/
```

Comment:
Is it possible to save  Properties list with updated values?

Reply:
No, I removed this functionality. It was too much work to keep this functionality and it is not necessary as it already exists elsewhere (analogous to the saving of a intensity). You can always reset the state of your model, by applying a fit result to a model.